.travis.yml

```diff
@@ -15,12 +15,14 @@ before_script:
 script:
   - |
     travis-cargo build &&
-    travis-cargo --skip 1.1.0 test &&
+    travis-cargo build --features=no_std &&
```

Comment:
Yeah, I started looking up how to do it with environment variables, but this is much simpler.